src/components/pages/LogIn.tsx

```diff
@@ -455,24 +455,24 @@ class LogInView extends React.Component<LogInProps, LogInState> {
           "phoneLogInStatus was an unexpected value:",
           this.props.phoneLogInStatus
         );
-      case PhoneLogInStatus.WAITING_FOR_PHONE_NUMBER_INPUT:
-      case PhoneLogInStatus.SENDING_PHONE_NUMBER:
-      case PhoneLogInStatus.SENDING_PHONE_NUMBER_FAILED:
+      case PhoneLogInStatus.WAITING_FOR_PHONE_INPUT:
+      case PhoneLogInStatus.SENDING_PHONE:
+      case PhoneLogInStatus.SENDING_PHONE_FAILED:
         return (
           <PhoneNumberForm
             waitingForCode={
               this.props.phoneLogInStatus.status ===
-              PhoneLogInStatus.SENDING_PHONE_NUMBER
+              PhoneLogInStatus.SENDING_PHONE
```

Comment:
Sending phone doesn't make much sense to me

Reply:
will change back